### ISSUE TITLE
Support Documentation: How to change the world seed?

### ISSUE THREAD

**Issue by oranges13** [Multiverse/Multiverse-Core#1981]
In the documentation, it says do not change the world seed unless you understand the consequences.

Our server has a monthly resource world that is reset every month, which includes changing the seed.

For the time being, our process is to shut off the server, delete the world files, and then we directly edit the worlds.yml file with the new seed on our FTP server. To avoid errors I would prefer to use in-game commands, specifically `/mv modify set` to make these changes, but it does not appear that includes updating the seed. If this requires unloading the world first while the server is running, that's fine as players will not be online (only administration staff) at the time.

Basically anything we can do to make the world reset faster, more effective, and more safe (less direct editing of the world.yml file) would be great.

**Comment by Tresswa** [comment]
I think you could delete the world with multiverse and create another one with the same name and a seed argument. You will probably have to reset any gamerules and multiverse world property options - but you could use commands.yml to make a pseudo-macro of these.

**Comment by benwoo1110** [comment]
ou cannot change the seed of a generated world, you need to create a new world or regen it with `/mvregen <worldname> -s <seed>`
